webapi/src/test/scala/org/knora/webapi/models/filemodels/FileModels.scala

```diff
@@ -45,9 +46,13 @@ sealed abstract case class UploadFileRequest private (
       case Some(v) => v
       case None    => FileModelUtil.getDefaultClassName(fileType)
     }
+    val resorceIRIOrNothing = resourceIRI match {
```

Comment:
Wouldn't `maybeResourceIri` better match our naming pattern or is there a reason why you used `...orNothing`?

Reply:
I prefer consistency for both cases. Because consistency let's us recognize similar constructs (which helps understanding code written by someone else) and also makes searching for certain constructs easier (although right now I don't know why I would want to search for all `maybe...` constructs - but you never know ;). This is just my opinion, maybe the others don't agree.

Reply:
I think the reason why I don't like "maybe" is because it's added at the beginning, so two variables don't look similar:

```scala
val something = "foo bar"
val maybeSomething = Some(something) // not clear on first glance that it's the same
val somethingOption = Some(something) // more clear on first glance that it's the same
```

Does that make sense? But I'm absolutely OK with "maybe" if you think it's good.

However, in this case it's not actually correct to call it `maybeResourceIRI` because it's not an option, it's either a string or an empty string, depending on the input. I'll rename it to `resourceIRIOrEmptyString`

Reply:
Yeah that'r the downside. From the other hand, if we use typing everywhere, we could skip naming properties like that, because type annotation strictly say what is this. `somethingMaybe` aka `callMeMaybe` ;) could be also another idea.

Reply:
I see your point. Still, it is the naming pattern we have right now and in my opinion it makes it easier to understand the code if the pattern is the same everywhere. We could still introduce a new naming pattern in V3. As the value here is not an option, you are actually right that it doesn't have to match the naming pattern of options. So, IMO you can as well call it `resourceIRIOrEmptyString`.